assets/app.js

```diff
@@ -5,7 +5,8 @@
  * (and its CSS file) in your base layout (base.html.twig).
  */
 
-import './bootstrap';
```

Comment:
this line should not be removed: this loads the boostrap of the assets management